styles/newsletter.scss

```diff
@@ -62,6 +62,7 @@ $o-grid-gutter-small: oGridGutter(S);
 }
 
 .n-newsletter-signup__description {
+	@include oTypographySerif($scale: 1);
```

Comment:
Out of interest did you find the font size/style somewhere? Some vague memory is saying it should be `$scale: 0` but very happy to be convinced otherwise.

Reply:
This was taken from `n-content-body` which was what it was inheriting beforehand. Happy to change it as it does seem a little large.

Reply:
Nope that's cool, just my dusty old memory